### PR TITLE
Fcstsprint/mdlcng defaults

### PR DESCRIPTION
**Pull request by HenryRWinterbottom**
**Description**

This PR contains default values for building the UFS `model_configure` file.

Some comments annotated with `HRW` are included requesting some further clarification for the default values.

**Type of change**

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

**How Has This Been Tested?**

A reduced `model_configure` has been built using the template and defined default values.
  
**Checklist**

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [] My changes need updates to the documentation. I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] New and existing tests pass with my changes
- [ ] Any dependent changes have been merged and published

### CODE REVIEW

File: ush/python/pygfs/ufswm/ufs.py [hunk 1]
```diff
@@ -104,6 +104,68 @@ def set_ufs_config(self) -> Dict[str, Any]:
...
+
+        """
+
+        cfg = AttrDict()
+
+        CDATE = datetime_to_YMDH(dt=self._config.CDATE)
+        cfg.SYEAR = CDATE[0:4]
+        cfg.SMONTH = CDATE[5:6]
+        cfg.SDAY = CDATE[7:8]
+        cfg.SHOUR = CDATE[9:10]
```
Comment (aerorahul) [inline ush/python/pygfs/ufswm/ufs.py]: I don't think you need `CDATE` at all.
```suggestion
        cfg.SYEAR = self._config.current_cycle.year
        cfg.SMONTH = self._config.current_cycle.month
        cfg.SDAY = self._config.current_cycle.day
        cfg.SHOUR = self._config.current_cycle.hour
```

File: ush/python/pygfs/ufswm/ufs.py [hunk 2]
```diff
@@ -104,6 +104,68 @@ def set_ufs_config(self) -> Dict[str, Any]:
...
+
+        cfg.FHMAX = self._config.FHMAX
+        cfg.RESTART_INTERVAL = self._config.restart_interval
+
+        # HRW: This is NoneType for now and will be constructed in a subsquent PR.
```
Comment (aerorahul) [inline ush/python/pygfs/ufswm/ufs.py]:
```suggestion
        # TODO: HRW: This is NoneType for now and will be constructed in a subsquent PR.
```
`TODO` shows up bright in an IDE.

**Review by aerorahul — APPROVED** [review]
Looks good.
This might conflict w/ some of the changes in #1496, but we can work it out.

File: ush/python/pygfs/ufswm/ufs.py [hunk 3]
```diff
@@ -8,6 +8,7 @@
 from pygw.template import Template, TemplateConstants
 from pygw.file_utils import FileHandler
 from pygw.logger import logit
+from pygw.timetools import datetime_to_YMDH
```
Comment (aerorahul) [inline ush/python/pygfs/ufswm/ufs.py:11]: Don't need this if you can access the year, month, day and hour directly from the datetime object.

Comment (HenryRWinterbottom) [reply ush/python/pygfs/ufswm/ufs.py:11]: Yep. I will remove in the next PR.

**Pull request merged**